dpsim/src/CMakeLists.txt

```diff
@@ -23,7 +24,9 @@ set(DPSIM_INCLUDE_DIRS
 	)
 
 set(DPSIM_LIBRARIES
-	dpsim-models)
+	dpsim-models
+	readerwriterqueue #TODO: Is there any need to only add this + the interface classes conditionally / introduce a new cmake variable for this?
```

Comment:
I think it is ok to add this unconditionally.

Reply:
I would agree. Removing the comment...